pkg/scheduler/service/bookkeeper_test.go

```diff
@@ -67,8 +52,11 @@ func TestBookkeeper(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second) //stop bookkeeper after 5 sec
 	defer cancel()
 
+	transition := newClusterStatusTransition(dbConn, inventory, reconRepo, logger.NewLogger(true))
 	start := time.Now()
-	require.NoError(t, bk.Run(ctx))
+	require.NoError(t, bk.Run(ctx,
+		markOrphanOperation{transition: transition, logger: transition.logger},
```

Comment:
I still not know the purpose, but why currently need to do two operation tasks in one bookkeeper run? Let's discuss in in our meeting.